src/features/duplicates/store.tsx

```diff
@@ -12,7 +12,7 @@ export interface ZetkinDuplicate {
 }
 
 export interface DuplicatesStoreSlice {
-  duplicatesList: RemoteList<ZetkinDuplicate>;
+  duplicatesList: RemoteList<PotentialDuplicate>;
 }
 
 const initialState: DuplicatesStoreSlice = {
```

Comment:
I think you should look over the names in the store so they also reflect the name change of the type - so that things named something with `duplicate/s` here is re-named to `potentialDuplicate/s`